pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -394,6 +394,10 @@ func (gce *GCECloud) EnsureTCPLoadBalancer(name, region string, externalIP net.I
 		PortRange:  fmt.Sprintf("%d-%d", minPort, maxPort),
 		Target:     gce.targetPoolURL(name, region),
 	}
+	if loadBalancerIP != nil {
+		req.IPAddress = loadBalancerIP.String()
```

Comment:
@anguslees we're getting close to "code complete" for v1.1 - if we get this PR in, will you be able to make time to test/fix openstack?

@artfulcoder we should be able to make the first-order change (use the new field instead of externalIPs[0])